api/src/main/kotlin/org/jetbrains/kotlinx/dl/dataset/embeddedDatasets.kt

```diff
@@ -262,13 +262,16 @@ public fun cifar10Paths(cacheDirectory: File = File("cache")): Pair<String, Stri
 }
 
 /** Path to the Dogs-vs-Cats dataset. */
-private const val DOGS_CATS_IMAGES_ARCHIVE: String = "datasets/catdogs/data.zip"
+private const val DOGS_CATS_IMAGES_ARCHIVE: String = "datasets/dogs-vs-cats/data.zip"
+
+/** Path to the Data.zip url to be downloaded from. */
+private const val DOGS_CATS_IMAGES_DOWNLOAD_URL: String = "https://kotlindl.s3.amazonaws.com/datasets/catdogs/data.zip"
```

Comment:
I recognized that we have the same problem with the paths for the small subset of this dataset.
Could you please do the same refactoring for the `DOGS_CATS_SMALL_IMAGES_ARCHIVE `constant and `dogsCatsSmallDatasetPath `method?

Reply:
Done, also extended the extraction logic.